data/plugins/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## How to contribute
+
+If you wish to recommend a plugin or framework not included in the list yet:
+
+1. create a `.json` file in [data/plugins/myplugin.json](myplugin.json) including plugins credentials as
```

Comment:
as a general rule I am using owner-repository.json as fileName, just to ensure in the future we don't have 2 plugins with same name

Reply:
Of course! Fixed in af87fabbc3e7fef4f068b17fa5e00b879ceff55f